docs/security.html

```diff
@@ -28,7 +28,172 @@
+        TpUAdb2FERMPLFsx06H0x+TULivcp7HbS5yrkV9bXZmv/FD98x76QxXrOq1WpQhY
+        YDeGDjH+XQkJ6ZxBVBZNJDIpCnfQpfzXAvryQ+cm8oXUsKidtHf4pLMYViXX6BWX
+        Oc2hX4rG+lC8/NXW+1zVvCr9To9fngzjMA0GCSqGSIb3DQEBCwUAA4IBAQBfyVse
+        RJ+ugiNlWg5trZscqH0tlocbnek4UuV/xis2eAu9l4EFOM5kRt5GmkGZRcM/zHF8
+        BRJwXbf0fytMQKSPFk8R4/NGDOlzoK+F7uXeJ0S2u/T29xk0u2i4tjvleq6OCphE
+        i9vdjM0E0Whf9SHRhOXirOYFX3cL775XwKdzKKRkk+AszFR+mRu90rdoaepQtgGh
+        9Kfwr4+6AU/dPtdGuomtBQqMxCzlrLd8EYhVVQ97wHIZ3sPvlM5PIhOJ/YHSBJIC
+        75eo/4acDxZ+j3sR5kcFulzYwFLgDYBaKH/w3mYCgTALeB1zUkX53NVizIvhUd69
+        XJO4lDSDtGOlfort
+        -----END CERTIFICATE-----
+        subject=/C=US/ST=CA/L=Santa Clara/O=org/OU=org/CN=Sriharsha Chintalapani
+        issuer=/C=US/ST=CA/L=Santa Clara/O=org/OU=org/CN=kafak/emailAddress=test@test.com
+            </pre>
+        If the certificate not showed up or if there are any other error messages than your keystores not setup properly.</li>
+
+        <li><h4>Configuring Kafka Clients</h4>h4>
+        SSL is supported only for new Kafka Producer & Consumer, the older API is not supported. The configs for SSL will be same for both producer & consumer.<br>
+        If client authentication is not required in the broker, then the following is a minimal configuration example:
+        <pre>
+        security.protocol = SSL
+        ssl.truststore.location = "/var/private/ssl/kafka.client.truststore.jks"
+        ssl.truststore.password = "test1234"
+            </pre>
+
+        If client authentication is required, then a keystore must be created like in step 1 and the following must also be configured:
+            <pre>
+        ssl.keystore.location = "/var/private/ssl/kafka.client.keystore.jks"
+        ssl.keystore.password = "test1234"
+        ssl.key.password = "test1234"
+                </pre>
+        Other configuration settings that may also be needed depending on our requirements and the broker configuration:
+            <ol>
+        <li>ssl.provider (Optional). The name of the security provider used for SSL connections. Default value is the default security provider of the JVM.)</li>
+        <li>ssl.cipher.suites (Optional). "A cipher suite is a named combination of authentication, encryption, MAC and key exchange algorithm used to negotiate the security settings for a network connection using TLS or SSL network protocol."</li>
+        <li>ssl.enabled.protocols=TLSv1.2,TLSv1.1,TLSv1 **Should list at least one of the protocols configured on the broker side**ssl.truststore.type = "JKS"</li>
+        <li>ssl.keystore.type = "JKS"</li>
+                </ol>
+<br>
+        Examples using console-producer and console-consumer:
+        <pre>
+            kafka-console-producer.sh --broker-list localhost:9093 --topic test --new-producer --producer-property "security.protocol=SSL"  --producer-property "ssl.truststore.location=client.truststore.jks" --producer-property "ssl.truststore.password=test1234"
```

Comment:
I think we can just use one --producer-property with multiple property key-values separated by comma?